internal/rps/rps.go

```diff
@@ -27,15 +27,9 @@ func NewAMTActivationServer(url string) AMTActivationServer {
 	return amtactivationserver
 }
 
-func PrepareInitialMessage(flags *rpc.Flags) Message {
-	//create activation request
+func PrepareInitialMessage(flags *rpc.Flags) (Message, error) {
```

Comment:
need unit tests for this

Reply:
I can't figure out a way to test this as written, the calls in that function are covered under message_test.go, would need to add an interface capability probably? could move it into the executor code .makeItSo() but that has no test coverage currently either.